utils/index.js

```diff
@@ -208,3 +208,67 @@ module.exports.processMarkdown = require('markdown-it')({
   linkify: true,
   typographer: true
 })
+
+/**
+ * Returns an array of express Router instances
+ * or []
+ */
+module.exports.loadThemeRoutes = function (app) {
+  console.log('Loading configured theme routes...')
+  const config = require('../config')
+  const path = require('path')
+  
+  try {
+    const themes = config.get('CKAN_THEME_ROUTES')
+    const themePath = config.get('CKAN_THEME_PATH') || './themes'
+    
+    if (!themes) return
+
+    return themes.split(' ').forEach(theme => {
+      const resource = path.join(process.cwd(), themePath, theme, 'routes.js')
```

Comment:
@starsinmypockets I think this can fail in systems other than unix-based but let's leave it with some comment.